### PR TITLE
Add Microsoft.Net.RoslynDiagnostics.props file to add analyzer assemb…

### PR DESCRIPTION
**Pull request by mavasani**
…lies to Roslyn projects - this props file gets pulled into Settings.Targets which is imported by all projects in Roslyn.sln.

### CODE REVIEW

**Comment by mavasani** [comment]
Tagging @dotnet/roslyn-analysis for review.

File: src/AggregateNugetPackages/Microsoft.Net.RoslynDiagnostics/NuGet/Microsoft.Net.RoslynDiagnostics.props [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition="'$(ProjectLanguage)' == 'CSharp'">
+    <Analyzer Include="$(NuGetPackageRoot)\Microsoft.ApiDesignGuidelines.Analyzers\$(RoslynDiagnosticsNugetPackageVersion)\analyzers\dotnet\cs\Analyzer.Utilities.dll" />
```
Comment (mavasani) [inline src/AggregateNugetPackages/Microsoft.Net.RoslynDiagnostics/NuGet/Microsoft.Net.RoslynDiagnostics.props]: @lgolding We will need to add entries for C# and VB specific utility assemblies here once you have merged your change which adds these assemblies.

Comment (lgolding) [inline src/AggregateNugetPackages/Microsoft.Net.RoslynDiagnostics/NuGet/Microsoft.Net.RoslynDiagnostics.props]: Analyzer.Utilities doesn't contain analyzers. Should this be here?

Comment (lgolding) [inline src/AggregateNugetPackages/Microsoft.Net.RoslynDiagnostics/NuGet/Microsoft.Net.RoslynDiagnostics.props]: (Just saw your note -- yes -- assuming these belong here at all.)

Comment (mavasani) [inline src/AggregateNugetPackages/Microsoft.Net.RoslynDiagnostics/NuGet/Microsoft.Net.RoslynDiagnostics.props]: As per the current design, we also need to specify dependencies of analyzer assemblies as analyzers in the project file. This is how VS live analysis knows to shadow copy the dependencies next to actual analyzer assemblies.

**Comment by lgolding** [comment]
:+1:

**Pull request merged**